light_curves/multiband_lc.ipynb

```diff
@@ -74,34 +74,43 @@
     "import sys\n",
     "import time\n",
     "import warnings\n",
-    "from math import ceil\n",
     "from multiprocessing import Pool\n",
+    "from pathlib import Path\n",
     "\n",
     "import astropy.units as u\n",
-    "import matplotlib.pyplot as plt\n",
-    "import numpy as np\n",
-    "from astropy.table import Table\n",
-    "from scipy import stats\n",
     "import pandas as pd\n",
     "\n",
     "warnings.filterwarnings('ignore')\n",
     "\n",
     "# local code imports\n",
-    "sys.path.append('code/')\n",
-    "from panstarrs import panstarrs_get_lightcurves\n",
+    "sys.path.append('code_/')\n",
+    "from data_structures import MultiIndexDFObject\n",
     "from gaia_functions import Gaia_get_lightcurve\n",
     "from HCV_functions import HCV_get_lightcurves\n",
-    "from icecube_functions import icecube_get_lightcurve\n",
-    "from sample_selection import get_lamassa_sample, get_macleod16_sample, get_ruan_sample, get_macleod19_sample, get_sheng_sample, \\\n",
-    "    get_green_sample, get_lyu_sample, get_lopeznavas_sample, get_hon_sample, get_yang_sample,  get_SDSS_sample, clean_sample\n",
-    "from data_structures import MultiIndexDFObject\n",
     "from heasarc_functions import HEASARC_get_lightcurves\n",
-    "from TESS_Kepler_functions import TESS_Kepler_get_lightcurves\n",
+    "from icecube_functions import icecube_get_lightcurve\n",
+    "from panstarrs import panstarrs_get_lightcurves\n",
     "from plot_functions import create_figures\n",
+    "from sample_selection import (clean_sample, get_green_sample, get_hon_sample, get_lamassa_sample, get_lopeznavas_sample,\n",
+    "    get_lyu_sample, get_macleod16_sample, get_macleod19_sample, get_ruan_sample, get_SDSS_sample, get_sheng_sample, get_yang_sample)\n",
+    "from TESS_Kepler_functions import TESS_Kepler_get_lightcurves\n",
     "# Note: WISE and ZTF data are temporarily located in a non-public AWS S3 bucket. It is automatically\n",
     "# available from the Fornax SMCE, but will require user credentials for access outside the SMCE.\n",
     "from WISE_functions import WISE_get_lightcurves\n",
-    "from ztf_functions import ZTF_get_lightcurve\n"
+    "from ztf_functions import ZTF_get_lightcurve"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a \"data\" directory to store any data that needs to be downloaded\n",
+    "Path(\"data\").mkdir(exist_ok=True)\n",
+    "\n",
+    "# create an \"output\" directory to store any results that we want to persist\n",
+    "Path(\"output\").mkdir(exist_ok=True)"
```

Comment:
I added this cell at the top of the notebook to create the "data" and "output" directories up front. However, I wonder if it would be better to check those two (empty) directories into the repo to avoid doing this in the notebook? I would still leave those two directories in the .gitignore file so that nothing gets accidentally committed to them. This would reduce clutter in the notebook and simplify the workflow (perhaps especially helpful if/when we start porting code out of the notebook to run large jobs). I'm thinking about this in the context of all future notebooks, not just this one. It would be nice to use the same process for all of them, and make it as simple as possible.

Reply:
@bsipocz what is your opinion on this?  I see both of Troy's points.  1) declutter so not have it in the notebook but 2) make sure the user sees the organizational tools that we have setup, so leave in the notebook.  I lean slightly towards not having it in the notebook since the SP has a nice file structure list on the left column so the directories are quite transparent.